src/main/resources/static/js/jquery.confirm.js

```diff
@@ -144,4 +144,75 @@
         cancelButtonClass: "btn-default",
         dialogClass: "modal-dialog"
     }
+
+    $.alert = function (options, e) {
+        // Do nothing when active confirm modal.
+        if ($('.confirmation-modal').length > 0)
+            return;
+
+        // Parse options defined with "data-" attributes
+        var dataOptions = {};
+        if (options.button) {
+            var dataOptionsMapping = {
+                'title': 'title',
+                'text': 'text',
+                'dialog-class': 'dialogClass'
+            };
+            $.each(dataOptionsMapping, function(attributeName, optionName) {
+                var value = options.button.data(attributeName);
+                if (value) {
+                    dataOptions[optionName] = value;
+                }
+            });
+        }
+
+        // Default options
+        var settings = $.extend({}, $.confirm.options, {
+            confirm: function () {
+                var url = e && (('string' === typeof e && e) || (e.currentTarget && e.currentTarget.attributes['href'].value));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=javascript%3AS1067)